pkg/appmanager/as3Manager.go

```diff
@@ -833,6 +840,37 @@ func (appMgr *Manager) getUnifiedAS3Declaration(as3CfgmapDecl as3Declaration, ro
 	if err != nil {
 		log.Debugf("[AS3] Unified declaration: %v\n", err)
 	}
+
+	// Override AS3 Config if AS3 Override is enabled
+	if appMgr.OverrideAS3Decl != "" {
+		// Fetch configMap's namespace and name
+		overrideAS3Decl := strings.Split(appMgr.OverrideAS3Decl, "/")
+		// Fetch configMap using namespace and name as an input
+		cfgMap, err := appMgr.getConfigMapUsingNamespaceAndName(overrideAS3Decl[0], overrideAS3Decl[1])
+		if err != nil {
+			log.Errorf("[AS3] ConfigMap not found for this name: %v and namespace: %v, error: %v",
```

Comment:
ConfigMap '%v' not found in namespace '%v'. Error: %v.

Reply:
Done